### PR TITLE
Update the changelog for 5.2.3 version

### PR DESCRIPTION
**Pull request by mzorai**

### CODE REVIEW

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 1]
```diff
@@ -8,41 +8,37 @@ eXo ECMS ${org.exoplatform.platform.version}
...
+    * [PLF-8220] - Creating Task projects: Primary_key violation error
+    * [PLF-8346] - Document App: deleting default folder Symlink doesn't work 
+    * [PLF-8347] - Document App: default folder Symlink cut/paste doesn't work
+    * [PLF-8406] - [MOB] Sharing a photo in eXo mobile app from galery breaks the activity stream
+    * [PLF-8408] - Unified search similar types' value issue
+    * [PLF-8414] - Unable to edit a user without email
+    
+
+** Improvement
+    * [PLF-8380] - [MOB] You can't comment directly from activity stream
```
Comment (boubaker) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: I changed the description of issue to `Can't comment directly from activity stream on mobile`

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 2]
```diff
@@ -8,41 +8,37 @@ eXo ECMS ${org.exoplatform.platform.version}
...
-    * [PLF-8339] - When applying filters in the leftNavigationMenu, "Space not found" page is displayed when clicking on a space which name has been edited
-    * [PLF-8289] - Browser Console error when using Ckeditor (Commons-Editor.js)
-    * [PLF-8175] - Display problem for arabic language in the document thumbnail in the activity stream
-    * [PLF-8156] - Can not insert image in web content under IE11 with special characters folders
-    * [PLF-8370] - Wrong Users status in the chat room page
+    * [PLF-7990] - empty page when displaying more than 1000 users in people page
+    * [PLF-8220] - Creating Task projects: Primary_key violation error
+    * [PLF-8346] - Document App: deleting default folder Symlink doesn't work 
+    * [PLF-8347] - Document App: default folder Symlink cut/paste doesn't work
+    * [PLF-8406] - [MOB] Sharing a photo in eXo mobile app from galery breaks the activity stream
```
Comment (boubaker) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: I changed the description of issue to `Sharing a photo from mobile app breaks the activity stream`

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 3]
```diff
@@ -8,41 +8,37 @@ eXo ECMS ${org.exoplatform.platform.version}
...
+    * [PLF-8408] - Unified search similar types' value issue
+    * [PLF-8414] - Unable to edit a user without email
+    
+
+** Improvement
+    * [PLF-8380] - [MOB] You can't comment directly from activity stream
+    * [PLF-8411] - [MOB] Old preview popup for images shared from mobile
```
Comment (boubaker) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: I changed the issue description to `Use new images preview when sharing it from mobile app`

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 4]
```diff
@@ -8,41 +8,37 @@ eXo ECMS ${org.exoplatform.platform.version}
...
 
-** Improvement
-    * [SOC-6239] - Make the identities sort field extensible
     
 eXo Calendar ${org.exoplatform.platform.version}
 *************************
 
 ** Bug
-    * [CAL-1361] - Nothing appears on Calendar when creating a task with views Board
+    * [CAL-1464] - Space's Calendar preferences
```
Comment (boubaker) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: I changed description to `Ignore selected calendars user preference in space calendar app`

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt [hunk 5]
```diff
@@ -8,41 +8,37 @@ eXo ECMS ${org.exoplatform.platform.version}
...
 
 ** Bug
-    * [PLF-8386] - Page Template's Access Permissions overridden
-    * [PLF-8344] - Document App: Remove actions for Default Folders
-    * [PLF-8339] - When applying filters in the leftNavigationMenu, "Space not found" page is displayed when clicking on a space which name has been edited
-    * [PLF-8289] - Browser Console error when using Ckeditor (Commons-Editor.js)
-    * [PLF-8175] - Display problem for arabic language in the document thumbnail in the activity stream
-    * [PLF-8156] - Can not insert image in web content under IE11 with special characters folders
-    * [PLF-8370] - Wrong Users status in the chat room page
+    * [PLF-7990] - empty page when displaying more than 1000 users in people page
```
Comment (boubaker) [inline plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: I changed description to `[Oracle DB] empty page when displaying more than 1000 users in people page`

**Review by boubaker — CHANGES_REQUESTED** [review]
Missing issues:

* JCR-2469
* PLF-8320
* SOC-6261
* PLF-8420

File: plf-packaging-resources/src/main/resources/CHANGE_LOG.txt (hunk 1 above)
Comment (mzorai) [reply plf-packaging-resources/src/main/resources/CHANGE_LOG.txt]: (y)

**Comment by mzorai** [comment]
> Missing issues:
>     * PLF-8320
PLF-8320 is not validated yet so I don't add it's ID until it is validated

**Review by boubaker — APPROVED** [review]

**Pull request merged**